nodejs/FatBeaconPeripheral/FatBeacon.js

```diff
@@ -0,0 +1,54 @@
+var bleno = require('bleno');
+var eddystoneBeacon = require('eddystone-beacon');
+
+/***********************Altering Library Functions**************************/
+/**
+ * To make this implementation compatible with existing nodjs libraries,
```

Comment:
I'd prefer something more succinct, like: In order to broadcast in the fatbeacon format, we need to override several functions in the advertisement-data module of the eddystone-beacon library.